docs/sphinx/readthedocs_build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
Nit: using `set -euo pipefail` would be good here, but the impl looks good.

Reply:
Done